Automattic/wc-calypso-bridge
# Re-enable WCPay menu in payments settings

Fixes https://github.com/automattic/wc-calypso-bridge/issues/707

This PR adds an admin notice in Payments settings page that contains a link to the welcome page with a param to trigger re-enable of the menu. The re-enable is done via `wc-admin/options` API. It also adds a URL utility function that's copied from wc-admin.

I used an admin note because I can't think of any better way yet, suggestions are welcome. Note the texts used are copied from existing strings available. I expect to change this if required.

![image](https://user-images.githubusercontent.com/3747241/125233395-0606f300-e311-11eb-97f3-1c2a194939dc.png)

### Testing instructions

1. Go to the WCPay welcome page, click on "No thanks" and remove the menu.
2. Go to Settings > Payments.
3. Observe an admin note is added per screenshot attached.
4. Click on "Learn more".
5. It should navigate back to the welcome page.
6. Refresh the page to observe that the WCPay menu is re-added on admin menu.
